templates/around/form_spot.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'layout.html' %}
+
+{% block title %}Add a new spot{% endblock %}
+
+{% block extra_css %}
+	<link rel=stylesheet href={{ url_for('static', filename='css/form-spot.css') }}>
```

Comment:
why so many spaces for indent in this whole file? :o

Reply:
It's really weird because it's clean on my editor and it's the same thing with the other files

Reply:
could you check on your side?

Reply:
It was tabs instead of spaces.